src/nl/surf/eduhub_rio_mapper/cli.clj

```diff
@@ -193,6 +194,10 @@
                                 ;; xml response converted to edn.
                            (assoc :attributes {:opleidingseenheidcode opleidingseenheidcode})
 
+                           (and (= :done status)
+                                (-> data :aanleveren_aangebodenOpleiding_response))
+                           (assoc :attributes {:aangebodenopleidingcode (str "<" id " (" type "Id)>")})
```

Comment:
willen we niet gewoon `id` hier ipv letterlijk `< id (bla bla bla)>` ? check dit met jelmer?

Reply:
Inderdaad gewoon `id`